v1/src/lib.rs

```diff
@@ -443,7 +443,7 @@ impl Storage for InfluxDbStorage {
         // encode the value as a string to be stored in InfluxDB, converting to base64 if the buffer is not a UTF-8 string
         let (base64, strvalue) = match payload.try_to_string() {
             Ok(s) => (false, s),
-            Err(err) => (true, b64_std_engine.encode(err.to_string()).into()),
+            Err(err) => (true, b64_std_engine.encode(payload.to_bytes()).into()),
```

Comment:
Shall be `Err(_)` instead of `Err(err)` , otherwise rustc complains that `err` id not used.

Reply:
doh! indeed, ill get that sorted, thanks